.ci-scripts/release/start-a-release.bash

```diff
@@ -28,13 +27,16 @@ source "${base}/config.bash"
 # We validate all that need to be set in case, in an absolute emergency,
 # we need to run this by hand. Otherwise the GitHub actions environment should
 # provide all of these if properly configured
-if [[ -z "${GITHUB_ACTOR}" ]]; then
-  echo -e "\e[31mName of the user to make changes to repo as need to be set in GITHUB_ACTOR. Exiting."
-  exit 1
-fi
-
-if [[ -z "${GITHUB_TOKEN}" ]]; then
-  echo -e "\e[31mA personal access token needs to be set in GITHUB_TOKEN. Exiting."
+if [[ -z "${RELEASE_TOKEN}" ]]; then
+  echo -e "\e[31mA personal access token needs to be set in RELEASE_TOKEN."
+  echo -e "\e[31mIt should not be secrets.GITHUB_TOKEN. It has to be a"
```

Comment:
Should this be here?

Reply:
Yes. Definitely yes.

Reply:
The default assumption of someone who knows GH actions would be to use secrets.GITHUB_TOKEN. That is what I was doing. It won't work if you do.